language/generated-samples/v1/language_sentiment_text.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "analyze_sentiment")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+import sys
+
+# [START language_sentiment_text]
+
+from google.cloud import language_v1
```

Comment:
can this be `from google.cloud import language_v1 as language`?

Reply:
Are we also trying to move the imports to inside the code snippet and inside the method?

Not sure what's possible with the auto-gen tool.

Reply:
> Are we also trying to move the imports to inside the code snippet and inside the method

Either! There's a current project to decide on this moving forward. But, that aside, we should **prefer imports > no imports**. We can put them into the method or outside of the method.

#### Why do some of the current Python samples have the `import`s inside the method?

 - because the hand-written snippets are usually in a file alongside many other snippets[.py](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/language/cloud-client/v1/snippets.py)
 - we generate snippets in their own files
 - this means that we can more easily show imports!
 - this ALSO means that we can do this for COMPILED languages as well now!!! 🤘

Reply:
I'll update this to `as language` 😄

Reply:
@dizcology Is this a blocker or can we revisit after this sample?